### PR TITLE
Add missing mdn_url for Window.captureEvents()

### PR DESCRIPTION
**Pull request by teoli2003**
We documented this long time ago.

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -659,6 +659,7 @@
       },
       "captureEvents": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/captureEvents"
```
Comment (teoli2003) [inline api/Window.json:662]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/captureEvents",
```

**Review by Elchi3 — COMMENTED** [review]
Should standard_track be false?

**Comment by teoli2003** [comment]
Yes, done. It is defined in the spec but more for compatibility purpose than because they want it.

**Comment by Elchi3** [comment]
CI failure

> api.Window.captureEvents - Error → Marked as non-standard, but has a spec_url

I think the spec_url can be removed because it is not defining anything.

**Comment by teoli2003** [comment]
The spec is actually saying it should exists and do nothing. "Doing nothing" is actually defining some action. This makes this feature on the standard track (but useless).

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**